packages/bodiless-core/__tests__/hoc.test.tsx

```diff
@@ -51,15 +51,3 @@ describe('withLocalContextMenu', () => {
     expect(withMenu.name()).toEqual('WithLocalContextMenu');
   });
 });
-
-describe('withNodeDataHandlers', () => {
-  it('should have componentData', () => {
-    const values = {
-      some: Math.random(),
-    };
-    const data = observable(values);
```

Comment:
Maybe we should put this test back and just remove the `observable` call (not sure what that is for), but i think the goal is to test that providing default data passes that to the underlying component?

Reply:
This test is now part of @bodiless/data https://github.com/wodenx/Bodiless-JS/pull/72/files/e724f23a4708d0e15caadd4469ae12a52c7ed8e8#diff-c7826c3a64d9c44e725cd9028d80f9fe9a5aa4e29f1da59c37621fa669c3a006